pkg/proxy/ipvs/proxier.go

```diff
@@ -1992,7 +1992,7 @@ func (proxier *Proxier) syncEndpoint(svcPortName proxy.ServicePortName, onlyNode
 			klog.Errorf("Failed to parse endpoint: %v, error: %v", ep, err)
 			continue
 		}
-		portNum, err := strconv.Atoi(port)
+		portNum, err := utilnet.ParsePort(port, false)
```

Comment:
0 was previously allowed here, right? I'll defer to the networking/ipvs folks to answer whether a 0 port should be treated as an error at this point now

Reply:
Okay. I will wait on their comment

Reply:
@andrewsykim This seems correct (not zero)